### ISSUE TITLE
 Cannot find plugin.xml for plugin 'package'

### ISSUE THREAD

**Issue by OmniaGM** [axemclion/react-native-cordova-plugin#11]
I follow the instruction in this blog http://blog.nparashuram.com/2015/10/using-cordova-plugins-in-react-native.html and this is the result.

``` r, engine='bash', count_lines
$ node node_modules/react-native-cordova-plugin/cli/cli.js add cordova-plugin-dialog

Adding plugins cordova-plugin-dialog
Initialized Android resources

Users/me/project/node_modules/q/q.js:155
                throw e;
                ^
CordovaError: Cannot find plugin.xml for plugin 'package'. Please try adding it again.
    at new PluginInfo (Users/me/project/node_modules/cordova-lib/node_modules/cordova-common/src/PluginInfo/PluginInfo.js:338:15)
    at PluginInfoProvider.get (Users/me/project/node_modules/cordova-lib/node_modules/cordova-common/src/PluginInfo/PluginInfoProvider.js:35:32)
    at Users/me/project/node_modules/cordova-lib/src/plugman/fetch.js:150:51
    at _fulfilled (Users/me/project/node_modules/cordova-lib/node_modules/q/q.js:787:54)
    at self.promiseDispatch.done (Users/me/project/node_modules/cordova-lib/node_modules/q/q.js:816:30)
    at Promise.promise.promiseDispatch (Users/me/project/node_modules/cordova-lib/node_modules/q/q.js:749:13)
    at Users/me/project/node_modules/cordova-lib/node_modules/q/q.js:557:44
    at flush (Users/me/project/node_modules/cordova-lib/node_modules/q/q.js:108:17)
    at doNTCallback0 (node.js:417:9)
    at process._tickCallback (node.js:346:13)

```

Any suggestions!

**Comment by axemclion** [comment]
What command are you trying to run, and what is the plugin you are trying to add ?

**Comment by OmniaGM** [comment]
this is the command 

```
 node node_modules/react-native-cordova-plugin/cli/cli.js add cordova-plugin-dialog
```

and I'm trying to add dialog plugin.

**Comment by axemclion** [comment]
Did you mean to install `cordova-plugin-dialogs` ? Note the plural. The `cordova-plugin-dialog` seems to be an empty npm package pointing to a [repo](https://github.com/phi-jp/cordova-plugin-dialog), with just a README and a package.json.

**Comment by axemclion** [comment]
@OmniaGM Was this resolved ?

**Comment by axemclion** [comment]
@OmniaGM  I just published 1.0.0 version of this plugin on npm. I also now have an [example app](https://github.com/axemclion/react-native-cordova-plugin/tree/examples) that shows how to use the plugins. 

Closing the issue - if you still find this to be an issue, please re-open it.
